lighthouse-cli/commands/commands.js

```diff
@@ -5,10 +5,5 @@
  */
 'use strict';
 
-const listAudits = require('./list-audits.js');
-const listTraceCategories = require('./list-trace-categories.js');
-
-module.exports = {
-  listAudits,
-  listTraceCategories,
-};
+export {listAudits} from './list-audits.js';
```

Comment:
complete nit, but not sure about this style. Maybe re-exporting is awkward no matter what, but I do like the explicitness of the other way